compiler/test/dotty/tools/dotc/vulpix/Statuses.java

```diff
@@ -0,0 +1,20 @@
+package dotty.tools.dotc.vulpix;
+
+import java.io.Serializable;
+
+/** The status of each call to `main` in the test applications */
+public class Statuses {
+    interface Status {}
+
+    static class Success implements Status, Serializable {
+        public final String output;
+        public Success(String output) { this.output = output; }
+    }
+
+    static class Failure implements Status, Serializable {
+        public final String output;
+        public Failure(String output) { this.output = output; }
+    }
+
+    static class Timeout implements Status, Serializable {}
```

Comment:
Is this something that a child JVM can send? Otherwise it should not be part of `Status` assuming `Status` defines the communication protocol between the JVMs.